images/macos/provision/utils/utils.sh

```diff
@@ -171,3 +171,26 @@ configure_user_tccdb () {
     local sqlQuery="INSERT OR IGNORE INTO access VALUES($values);"
     sqlite3 "$dbPath" "$sqlQuery"
 }
+
+get_github_package_download_url() {
+    local REPO_OWNER=$1
+    local REPO_NAME=$2
+    local FILTER=$3
+    local VERSION=$4
+    local API_PAT=$5
+    local SEARCH_IN_COUNT="100"
+
+    [ $API_PAT ] && authString="-H 'Authorization: token $API_PAT'"
```

Comment:
This condition doesn't work. I think we should check is string is not empty with -n flag.